.github/workflows/visual-regression-tests.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  visual-regression-test-components:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true # Also runs `bundle install`
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+- name: Get Yarn cache directory path
+  id: yarn-cache-directory-path
+  run: echo "::set-output name=dir::$(yarn cache dir)"
+- name: Check for cached Yarn dependencies
+  uses: actions/cache@v2
+  with:
+    path: ${{ steps.yarn-cache-directory-path.outputs.dir }}
+    key: yarn-${{ hashFiles('**/yarn.lock') }}
+    restore-keys: yarn-
+- run: yarn install --frozen-lockfile
+- uses: percy/exec-action@v0.3.1
+  with:
+    command: bundle exec rspec --tag visual_regression
+  env:
+    PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
```

Comment:
Could this token be added at an organisation level, since updating it could be a faff once it's being used in 2+ repositories?

Reply:
[Seems we can ](https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-secrets-and-runners-with-your-organization#sharing-secrets-within-an-organization) but I'll need someone with The Power like from @alex-ju 🙏

Reply:
Sure, I'll catch-up with Kelvin on setting this up at the org level